frappe/docs/user/en/tutorial/start.md

```diff
@@ -24,7 +24,7 @@ Password : **Use the password that was created during installation**
 
 When you login, you should see the "Desk" home page
 
-<img class="screenshot" alt="Desk" src="{{docs_base_url}}/assets/img/desk.png">
+<img class="screenshot" alt="Desk" src="http://imgur.com/Zmje1Qn.png">
```

Comment:
@ratskin,

can you add the updated images in /frappe-bench/apps/frappe/frappe/docs/assets/img location instead of imgur.com?

Also please use the cropped images instead of full-screen images